test/steps/test_step.py

```diff
@@ -370,7 +375,9 @@ class StepTest(Step):
     with open(report_file) as f:
         actual_report_data = f.read()
 
-    assert out_metrics == new_metric, ".run_subprocess() generated invalid metrics"
+    assert (
+        out_metrics == subprocess_result
```

Comment:
shouldn't be named `out_metrics` anymore but I guess it's up to you if you want to change it.